mteb/abstasks/MultilingualTask.py

```diff
@@ -1,10 +1,12 @@
 from __future__ import annotations
 
+from abc import ABC
+
 from .AbsTask import AbsTask
 from .MultiSubsetLoader import MultiSubsetLoader
 
 
-class MultilingualTask(MultiSubsetLoader, AbsTask):
+class MultilingualTask(MultiSubsetLoader, AbsTask, ABC):
```

Comment:
Not sure why this is here?

Reply:
`MultilingualTask` is only used as a parent class for multilingual classes, so I thought it could be an ABC class to avoid issues with abstract methods. However, if it doesn't fit, I can remove it.

Reply:
Ahh right, so no functional change? In that case I would probably remove it.